recipes/dtc/conda-forge.yml

```diff
@@ -0,0 +1,2 @@
+win:
+  enabled: False
```

Comment:
I couldn't remember whether I need to do this manually here or if it happens because I have `build.skip: True  # [win]` in my meta.yaml once we convert this into a feedstock.  I'm beginning to think I should delete this file before we merge this.